raiden-dapp/src/service-worker/assistant.ts

```diff
@@ -0,0 +1,28 @@
+/* istanbul ignore file */
+import { Store } from 'vuex';
+import compareVersions from 'compare-versions';
+import { CombinedStoreState } from '@/store/index';
+
+const VERSION_FILE_PATH = (process.env.BASE_URL ?? '/') + 'version.json';
+
+export default class ServiceWorkerAssistant {
+  constructor(private store: Store<CombinedStoreState>) {
+    setInterval(this.updateAvailableVersion, 1000 * 60);
```

Comment:
Once per minute seems quite often IMHO.

Reply:
That was the former used value. I just continued to use it. I'm totally open for any value you guys like to. I have no strong feelings about it. Please make a suggestion.
Though 1min is useful for testing. Maybe making it configurable would be helpful. The question would be how. Any suggestions?

Reply:
We have decided to change this to having an initial version check when the page loads and then every hour only. Going to implement this.

Reply:
Implemented. I adopted the test description of the PR for this.
As you have agreed with me on this solution in RC, I'm going to close this conversation since the implementation was trivial.